google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -457,15 +453,44 @@ def group_name(parent):
                 member_name=name)
             self.session.add(self.member_cache[member])
 
-        parent_group = group_name(parent)
+        parent_group = self.group_name(parent)
 
         if parent_group not in self.membership_map:
             self.membership_map[parent_group] = set()
 
         if member not in self.membership_map[parent_group]:
             self.membership_map[parent_group].add(member)
             self.membership_items.append(
-                dict(group_name=group_name(parent), members_name=member))
+                dict(group_name=self.group_name(parent), members_name=member))
+
+
+    def group_name(self, group):
```

Comment:
Nit, this doesn't need to be in the class, it can be a top level function since it doesn't use self.